log_outgoing_requests/models.py

```diff
@@ -86,6 +101,9 @@ class OutgoingRequestsLog(models.Model):
     class Meta:
         verbose_name = _("Outgoing Requests Log")
         verbose_name_plural = _("Outgoing Requests Logs")
+        permissions = [
+            ("can_view_logs", "Can view outgoing request logs"),
```

Comment:
I thought this was created automatically by Django? https://docs.djangoproject.com/en/3.2/topics/auth/default/#default-permissions